app/src/main/java/com/e/salesapp/CategoriesListFragment.java

```diff
@@ -0,0 +1,39 @@
+package com.e.salesapp;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import androidx.fragment.app.Fragment;
+
+public class CategoriesListFragment extends Fragment {
+
+    OnCategorySelected listener;
+
+    CategoriesListFragment(OnCategorySelected categorySelected)
+    {
+        this.listener = categorySelected;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View root = inflater.inflate(R.layout.fragment_categories_list, container, false);
```

Comment:
The name of your variable is no significant justo follow the name conventions of android